djangooidc/oidc.py

```diff
@@ -227,6 +227,11 @@ def create_client(self, userid="", **kwargs):
                 key_jar =keyio.KeyJar(verify_ssl=verify_ssl)
                 key_jar.add_kb("",key_bundle)
                 args["keyjar"] = key_jar
+            if "keyset_jwt_string" in kwargs["client_registration"].keys():
```

Comment:
A couple of naming concerns:
- Is this a `jwt` or a `jwk`?  The other key string says `keyset_jwk_file` but here you call it a `keyset_jwt_string`.
- Should we call it a `string` when it's a Python dict you're expecting/passing?

Reply:
Good points, let me correct. Originally it was a string and cruft :)

Reply:
Fixed!